index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="body-collection">
+      <header>
+        <div class="sign-in">
+          <div class="sign-in__already">
+            <div>Already have an Intuit account?</div>
+            <div><a>I forgot my User ID or Password</a></div>
+          </div>
+          <div class="sign-in__button">
+            <a class="sign-in-button"
+              ><img class="lock-img"
+                src="https://img.icons8.com/material-outlined/24/000000/lock.png"
```

Comment:
The `img` element must have an `alt` attribute. To understand why you can check this [article](https://dev.to/belhassen07/why-is-the-alt-attribute-so-important-for-images-c37).